aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -861,6 +861,47 @@ Tensor _nested_view_from_buffer(
     storage_offsets);
 }
 
+Tensor _nested_view_from_jagged(
+    const Tensor& values,
+    const Tensor& offsets,
+    const Tensor& dummy,
+    const c10::optional<Tensor>& lengths,
+    const int64_t ragged_idx) {
+  TORCH_INTERNAL_ASSERT(
+      false, "_nested_view_from_jagged(): expected to be implemented from Python");
+  return Tensor();
+}
+
+Tensor _nested_get_values(const Tensor& self) {
+  TORCH_INTERNAL_ASSERT(
+      false, "_nested_get_values(): expected to be implemented from Python");
+  return Tensor();
```

Comment:
fwiw, you don't need the returns here, TORCH_INTERNAL_ASSERT Is written so that the compiler knows afterwards is unreachable